src/sortStudents.ts

```diff
@@ -1,16 +1,52 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+const averageGrade = (grades: number[]): number => {
+  return grades.reduce((prev, next) => prev + next, 0) / grades.length;
+};
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  return [...students].sort((prev, curr) => {
```

Comment:
In this task better use more obvious variable names.
```suggestion
  return [...students].sort((firstStudent, secondStudent) => {
```